stock_quant_reservation_info/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to look who is reserving the products and check the following picking.
```

Comment:
Who ? Could you be a little bit more accurate ?

Reply:
Also changed, thanks again!